docs/source/using/troubleshooting.adoc

```diff
@@ -13,11 +13,11 @@ toc::[]
 This section contains solutions to common problems that you might
 encounter while using Minishift.
 
-[[troubleshooting-kvm]]
-== KVM driver
+[[troubleshooting-drivers]]
+== Troubleshooting problems with drivers
```

Comment:
I thought of putting general issues before the Driver issues similar to CDK docs, but then retained the existing order considering that drivers are prerequisites and an issue with them will prevent further use of Minishift. Also, the Driver issues seem like showstoppers and there are 5 of them, whereas we only have 1 general issue out of which only 1 is most likely to affect users often. If you think reversing the order is more useful, I will change it.